### ISSUE TITLE
Problem with install "trezor-gpg-pinentry-tk"

### ISSUE THREAD

**Issue by cwesystems** [romanz/trezor-agent#282]
When I run:
pip install trezor-gpg-pinentry-tk

gives me this error:
```
Collecting trezor-gpg-pinentry-tk
  Using cached https://files.pythonhosted.org/packages/65/9f/8821ca8a5a864127628fdcb3ca9da49f4e9aff44f7aab740602339ed0a29/trezor-gpg-pinentry-tk-0.0.7.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-3u2yboo1/trezor-gpg-pinentry-tk/setup.py", line 15, in <module>
        long_description=open('readme.md', 'r').read(),
    FileNotFoundError: [Errno 2] No such file or directory: 'readme.md'
```

Seems like the script needs "readme.md"

**Comment by romanz** [comment]
/cc @rendaw 
I'd suggest opening this issue at https://gitlab.com/rendaw/trezor-gpg-pinentry-tk.

Meanwhile, you can install this using:
```
pip install git+https://gitlab.com/rendaw/trezor-gpg-pinentry-tk
```

**Comment by rendaw** [comment]
@cwesystems Fixed now on pypi (version 0.0.9)
